sources/components/custom-styles/template-cs-content.scss

```diff
@@ -185,5 +185,217 @@
   */
   #{--content-link-hover-text-transform}: none;
 
+  /* VoogStyle
+    "pathI18n": ["product", "title"],
+    "titleI18n": "alignment",
+    "editor": "listPicker",
+    "list": {{ base_alignment_set }},
+    "scope": "global"
+  */
+  #{--content-area-product-title__text-alignment}: left;
+
+  /* VoogStyle
+    "pathI18n": ["product", "title"],
+    "titleI18n": "size",
+    "editor": "rangePicker",
+    "min": 8,
+    "max": 100,
+    "unit": "px",
+    "scope": "global"
+  */
+  #{--content-area-product-title__font-size}: 72px;
```

Comment:
Why so large font size? Let's use same like general h1 in design (32px)
Also line height is too high for this size.
![Screenshot 2022-02-03 at 17 40 25](https://user-images.githubusercontent.com/378755/152375933-67bcc447-2247-48d6-91e7-6dcae49573dd.png)


```suggestion
  #{--content-area-product-title__font-size}: 32px;
```